### PR TITLE
Add French Language

### PR DESCRIPTION
**Pull request by DoctaEnkoda**

### CODE REVIEW

**Comment by DoctaEnkoda** [comment]
Is this PR going to be set?

File: plugin/src/main/resources/locale/lang-fr.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+successfully-reset-map: <green>La carte a été réinitialisée avec succès pour le monde <world>
+paused-renders: <green>Rendu en pause pour le monde <world>
+unpaused-renders: <green>Reprise du rendu pour le monde <world>
+click-for-help: Cliquez pour demander l'aide
+plugin-reloaded: <green><name> v<version> rechargé
+command:
+  prefix: '<white>[<gradient:#C028FF:#5B00FF>Pl3xMap</gradient>]</white> '
+  message:
+    console-must-specify-player: <red>Vous devez spécifier un joueur pour exécuter
+      la commande par la console
```
Comment (granny) [inline plugin/src/main/resources/locale/lang-fr.yml:16]:
```suggestion
    console-must-specify-player: <red>Vous devez spécifier un joueur pour exécuter la commande par la console
```

File: plugin/src/main/resources/locale/lang-fr.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    player-not-found-for-input: <red>Aucun joueur n'a été trouvé avec les arguments '<input>'
+    console-must-specify-world: <red>Vous devez spécifier un monde pour exécuter
+      la commande par la console
```
Comment (granny) [inline plugin/src/main/resources/locale/lang-fr.yml:19]:
```suggestion
    console-must-specify-world: <red>Vous devez spécifier un monde pour exécuter la commande par la console
```

File: plugin/src/main/resources/locale/lang-fr.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    cancel-render: Annuler un rendu pour le monde spécifié
+    pause-render: Suspendre tous les rendus pour le monde spécifié
+    radius-render: Démarrer un rendu de rayon
+    hide: Masquer un joueur sur la carte
+    show: Montrer un joueur sur la carte
+  argument:
+    optional-world: Par défaut le monde actuel des joueurs s'il n'est pas fourni
+    optional-center: Par défaut à (<white>0<gray>,</gray> 0</white>) si non spécifié
+    optional-player: Par défaut le joueur en cours d'exécution si non spécifié (par la console
+      vous devez spécifier un joueur)
```
Comment (granny) [inline plugin/src/main/resources/locale/lang-fr.yml:39]:
```suggestion
    optional-player: Par défaut le joueur en cours d'exécution si non spécifié (par la console vous devez spécifier un joueur)
```

File: plugin/src/main/resources/locale/lang-fr.yml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    world-border: Bordure du Monde
+log:
+  prefix: <dark_aqua>[<light_purple>Pl3xMap</light_purple>]</dark_aqua>
+  started-full-render: <dark_aqua>Démarrage du rendu de la carte complète pour <yellow><world>
+  started-radius-render: <dark_aqua>Rendu de la carte de rayon démarré pour <yellow><world>
+  scanning-region-files: <yellow>Numérisation des fichiers de région... (this may take a moment)
+  found-total-region-files: <green>Fichiers Région <gray><total> <green>trouvé
+  finished-rendering: <dark_aqua>Carte de rendu terminée pour <yellow><world>
+  cancelled-rendering: <dark_aqua>Le rendu de la map <yellow><world> <dark_aqua>a été
+    intérrompu
```
Comment (granny) [inline plugin/src/main/resources/locale/lang-fr.yml:65]:
```suggestion
  cancelled-rendering: <dark_aqua>Le rendu de la map <yellow><world> <dark_aqua>a été intérrompu
```

File: plugin/src/main/resources/locale/lang-fr.yml [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  resumed-rendering: <dark_aqua>Le rendu de la map <yellow><world> <dark_aqua>a recontinuer
+  scanning-region-progress: <gray>(</gray><yellow><percent></yellow><gray>)</gray>
+    <dark_aqua>Monde<gray>:</gray> <yellow><world></yellow> Chunks<gray>:</gray> <yellow><current_chunks></yellow><gray>/</gray><yellow><total_chunks></yellow>
+    Temps Ecoulé<gray>:</gray> <yellow><elapsed></yellow> ETA<gray>:</gray> <yellow><eta></yellow>
+    Vitesse<gray>:</gray> <yellow><rate></yellow> cps
```
Comment (granny) [inline plugin/src/main/resources/locale/lang-fr.yml:70]:
```suggestion
  scanning-region-progress: <gray>(</gray><yellow><percent></yellow><gray>)</gray> <dark_aqua>Monde<gray>:</gray> <yellow><world></yellow> Chunks<gray>:</gray> <yellow><current_chunks></yellow><gray>/</gray><yellow><total_chunks></yellow> Temps Ecoulé<gray>:</gray> <yellow><elapsed></yellow> ETA<gray>:</gray> <yellow><eta></yellow> Vitesse<gray>:</gray> <yellow><rate></yellow> cps
```

File: plugin/src/main/resources/locale/lang-fr.yml [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+  scanning-region-progress-with-regions: <gray>(</gray><yellow><percent></yellow><gray>)</gray>
+    <dark_aqua>Monde<gray>:</gray> <yellow><world></yellow> Regions<gray>:</gray>
+    <yellow><current_regions></yellow><gray>/</gray><yellow><total_regions></yellow>
+    Chunks<gray>:</gray> <yellow><current_chunks></yellow><gray>/</gray><yellow><total_chunks></yellow>
+    Temps Ecoulé<gray>:</gray> <yellow><elapsed></yellow> ETA<gray>:</gray> <yellow><eta></yellow>
+    Vitesse<gray>:</gray> <yellow><rate></yellow> cps
```
Comment (granny) [inline plugin/src/main/resources/locale/lang-fr.yml:76]:
```suggestion
  scanning-region-progress-with-regions: <gray>(</gray><yellow><percent></yellow><gray>)</gray> <dark_aqua>Monde<gray>:</gray> <yellow><world></yellow> Regions<gray>:</gray> <yellow><current_regions></yellow><gray>/</gray><yellow><total_regions></yellow> Chunks<gray>:</gray> <yellow><current_chunks></yellow><gray>/</gray><yellow><total_chunks></yellow> Temps Ecoulé<gray>:</gray> <yellow><elapsed></yellow> ETA<gray>:</gray> <yellow><eta></yellow> Vitesse<gray>:</gray> <yellow><rate></yellow> cps
```

File: plugin/src/main/resources/locale/lang-fr.yml [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+    world-list: Mondes
+  layers:
+    player-tracker: Joueurs
+    world-spawn: Spawn
+    world-border: Bordure du Monde
+log:
+  prefix: <dark_aqua>[<light_purple>Pl3xMap</light_purple>]</dark_aqua>
+  started-full-render: <dark_aqua>Démarrage du rendu de la carte complète pour <yellow><world>
+  started-radius-render: <dark_aqua>Rendu de la carte de rayon démarré pour <yellow><world>
+  scanning-region-files: <yellow>Numérisation des fichiers de région... (this may take a moment)
```
Comment (granny) [inline plugin/src/main/resources/locale/lang-fr.yml:61]: (this may take a moment) isn't translated

**Review by granny — CHANGES_REQUESTED** [review]
A handful of formatting errors and a missing translation, otherwise LGTM.

I can't speak French so I can't really attest for the grammar and all that though. I'd like it if someone else who's fluent could look this over, but it's not required.

**Comment by DoctaEnkoda** [comment]
fixed

**Review by granny — APPROVED** [review]

**Pull request merged**